library/lib_location_constants.ks

```diff
@@ -1,34 +1,78 @@
+  kerbinLocations:add("VAB", Kerbin:GeoPositionLatLng(-0.0968, -74.6187)).
+  location_constants:add("launchpad",kerbinLocations["launchpad"]).
 
   // horizontal landing locations
-  "runway_09_start", Kerbin:GeoPositionLatLng(-0.0487, -74.7247),
-  "runway_09_end", Kerbin:GeoPositionLatLng(-0.0501, -74.4880),  // runway "lip"
-  "runway_27_start", Kerbin:GeoPositionLatLng(-0.0501, -74.4925),
-  "runway_27_end", Kerbin:GeoPositionLatLng(-0.0487, -74.7292),  // runway "lip"
-  "l1_runway_09_start", Kerbin:GeoPositionLatLng(-.0489, -74.7101),
-  "l1_runway_27_start", Kerbin:GeoPositionLatLng(-.0503, -74.5076),
-  "l2_runway_09_start", Kerbin:GeoPositionLatLng(-.0486, -74.7134),
-  "l2_runway_27_start", Kerbin:GeoPositionLatLng(-.0501, -74.5046),
-  "island_runway_09_start", Kerbin:GeoPositionLatLng(-1.5177, -71.9663),
-  "island_runway_27_start", Kerbin:GeoPositionLatLng(-1.5158, -71.8524),
-  "desert_runway_36_start", Kerbin:GeoPositionLatLng(-6.5998, -144.0407),
-  "desert_runway_18_start", Kerbin:GeoPositionLatLng(-6.4480, -144.0383)
-).
+  kerbinLocations:add("runway_09_start", Kerbin:GeoPositionLatLng(-0.0486, -74.7247)).
+  kerbinLocations:add("runway_09_overrun", Kerbin:GeoPositionLatLng(-0.0502, -74.4880)).  // runway "lip"
+  kerbinLocations:add("runway_27_start", Kerbin:GeoPositionLatLng(-0.0502, -74.4925)).
+  kerbinLocations:add("runway_27_overrun", Kerbin:GeoPositionLatLng(-0.0486, -74.7292)).  // runway "lip"
+  kerbinLocations:add("l1_runway_09_start", Kerbin:GeoPositionLatLng(-.0489, -74.7101)).
+  kerbinLocations:add("l1_runway_27_start", Kerbin:GeoPositionLatLng(-.0501, -74.5076)).
+  kerbinLocations:add("l2_runway_09_start", Kerbin:GeoPositionLatLng(-.0486, -74.7134)).
+  kerbinLocations:add("l2_runway_27_start", Kerbin:GeoPositionLatLng(-.0501, -74.5046)).
+  kerbinLocations:add("island_runway_09_start", Kerbin:GeoPositionLatLng(-1.5177, -71.9663)).
+  kerbinLocations:add("island_runway_27_start", Kerbin:GeoPositionLatLng(-1.5158, -71.8524)).
+  kerbinLocations:add("desert_runway_36_start", Kerbin:GeoPositionLatLng(-6.5998, -144.0409)).
+  kerbinLocations:add("desert_runway_18_start", Kerbin:GeoPositionLatLng(-6.4480, -144.0383)).
+
+  location_constants:add("runway_start",kerbinLocations["runway_09_start"]).
+  location_constants:add("reverse_runway_start",kerbinLocations["runway_27_start"]).
+
+  location_constants:add("kerbin",kerbinLocations).
+}
 
 // aliases
-set LocationConstants["runway_start"] to LocationConstants["runway_09_start"].
-set LocationConstants["reverse_runway_start"] to LocationConstants["runway_27_start"].
-
-for key in LocationConstants:keys {
-  if key:contains("desert") {
-    local alias to key:replace("desert", "dessert").
-    set LocationConstants[alias] to LocationConstants[key].
-  }.
-}.
+until not aliasing(location_constants).
+
+local function aliasing {
+  parameter locationConstants.
+  local didChange is false.
```

Comment:
I see little point to this change as if `didChange` is ever needed out side of the function then the more logical step would be to change how the function is called so that you can read the value in with something other than the until.

Reply:
It's not about this line specifically. I have a general policy never to use `is`. But I'm not going to make that more than a suggestion for anyone else.